tests/19-get-spanning-data.rs

```diff
@@ -16,12 +16,12 @@ fn main() {
         let mut new = ColorEntry::new();
```

Comment:
Isn't this a private method now?

Reply:
Still visible to the module in which `ColorEntry` is declared, all the tests go through just fine.